Lab1/lab1_polynomial.cpp

```diff
@@ -159,15 +159,19 @@ Polynomial::~Polynomial(){
 bool Polynomial::operator==(const Polynomial &target) {
     bool equals = true;
```

Comment:
can this be called something like `isEqual`? I think it would make the code more readable